packages/sui-studio/scripts/postinstall.js

```diff
@@ -0,0 +1,81 @@
+const fse = require('fs-extra')
+const fg = require('fast-glob')
+
+console.log(`Looking for test folder in ${process.env.INIT_CWD}`)
```

Comment:
Instead using always a environment variable, let's create a const here and reuse everywhere in case we want to recheck this in the future.

`const CWD = process.env.INIT_CWD`